culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+reviewed.
+
+A change can take many forms, for example:
+- Adding a new guide or piece of documentation
+- Adding a new module to the codebase
+- Modifying an existing guide or module
+
+
+## Participants in the Review Process
+
+### Contributor
+
+The contributor is the person proposing a change. Anyone can decide to be a
+contributor: the only requirements are that you have a change you need to
+propose, and that you agree to be a respectful participant in the review
+process. (When we have a code of conduct--link to it here.)
+
+### Reviewer
+
+The reviewers are the people who review the change, suggest changes to it, and
+eventually approve it. Anyone in the community can act as a reviewer: the only
+requirements are that you have some insight to share (or questions to ask)
+about the change, and that you agree to be a respectful participant in the
+process.
+
+### Maintainer
+
+The maintainers are responsible for the health and maintenance of the project,
+and have final say in the review process. Maintainers can block changes they
+think are harmful, or approve a change over others' objections. Someone can be
+a maintainer within a specific domain: for example, acting as maintainer for a
+particular repository, or even maintaining a particular guide. Since
+maintainers have special powers, they also have a special responsibility to act
+thoughtfully and fairly. New maintainers are chosen by the existing
+maintainers. In general, people who have a long track record of acting as
+contributors and reviewers may be good candidates for becoming maintainers.
+
+## Lifecycle of a Review
+
+### Proposal
+The first step is for a contributor to propose a _change_.
+The contributor needs to do two things at this stage:
+- Flesh out the change (e.g. writing the code), so that there is something to review
+- Explain the change (e.g. writing a pull request description), so that reviewers can easily understand it
+
+Both steps are vital, and contributors should put real effort into explaining the change well.
+Its not fair to ask a reviewer to puzzle through an undocumented change.
+
+Finally, before requesting a review, the contributor should _self-review_ their change.
+It's amazing how many issues can be spotted and fixed during self-review.
+This is respectful of reviewers' time, and makes the process a lot smoother.
+
+### Review
+
+Once a change is ready and has been self-reviewed, it's ready for review. Since
+most SourceCred artifacts are tracked via repositories on GitHub, this will
+usually mean creating a pull request.
+
+Once the change is proposed, anyone in the project can add their own review.
+The person proposing the change can also request specific reviewers, if certain
+individiuals have context or knowledge thet will make their reivew particularly
+valuable.
+
+On finishing their review, the reviewer can:
+- ask questions to better understand the change
+- suggest changes to it
+- approve the change
```

Comment:
Nit: prefer capitalizing for bullet points, consistent with the rest of the document.

Reply:
Fixed in update.